### PR TITLE
Add logstash support

### LINKED ISSUE #1321: add logstash to stack of laradock

### PR DESCRIPTION
**Pull request by blunket**
This PR adds support for Logstash. I have not updated documentation yet at this time as I want to ensure this is on the right track first.

This PR is a product of my first-ever experience with any of Docker, Laravel, Laradock OR Logstash, so I expect I may have made mistakes here. I would love any help and feedback to make this as perfect and portable as possible to help other devs. (Including help making anything into an environment variable that should be)

This fixes #1321 also.

##### I completed the 3 steps below:

- [x] I've read the [Contribution Guide](http://laradock.io/contributing).
- [ ] I've updated the **documentation**. (refer to [this](http://laradock.io/contributing/#update-the-documentation-site) for how to do so).
- [x] I enjoyed my time contributing and making developer's life easier :)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -697,7 +715,7 @@ services:
       networks:
         - frontend
         - backend
-
```
Comment (vladyslavstartsev) [inline docker-compose.yml]: can you please revert all of those whitespace removing lines back?
This PR is small, but it's definitely  easier to look at PR without such things

**Comment by vladyslavstartsev** [comment]
@blunket there is also not that much activity in past months

**Comment by blunket** [comment]
hey, thanks for the merge! I was expecting some changes would be necessary first. For example the people using logstash may not need MySQL. should we account for that or are we good for now?

**Comment by bestlong** [comment]
Just take up more storage space.
If you think the removal will be more appropriate.
new PR is welcome.

**Comment by DJviolin** [comment]
@blunket Please can you explain why the Logstash configuration needs to include `env_file` field explicitly?

It's creating an issue if you want to create a Laradock folder structure, where you keep it's repository untouched and calling docker-compose from a parent folder, it throws error because of file missing, for example:

```sh
$ docker-compose --env-file .env -f laradock/docker-compose.yml -f docker-compose.override.yml config up -d nginx mariadb adminer
```

Related issue: [https://github.com/laradock/laradock/issues/3146](https://github.com/laradock/laradock/issues/3146)

**Comment by DJviolin** [comment]
I think it would be much better to only include the needed environment variables for Logstash, like every other images. This way, it will be possible to use Laradock without to touch any file within the Laradock git repository and only copy out the .env file to the parent dir, where calling docker-compose is happening.

**Pull request merged**